metric_map.go

```diff
@@ -15,10 +15,14 @@ type MetricMap struct {
 	Timers   Timers
 	Gauges   Gauges
 	Sets     Sets
+
+	Forwarded bool
```

Comment:
Should this be public and not part of the above group?

Reply:
I believe it will eventually be consumed outside the package, it's been a while since I wrote the code that comes next, but I think that's the case.